composites/Plugin/Shared/components/YoastModal.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Modal from "react-modal";
+import styled from "styled-components";
+
+import colors from "../../../../style-guide/colors.json";
+import SvgIcon from "./SvgIcon";
+
+const StyledHeading = styled.h1`
+	float: left;
+	margin: -4px 0 2rem;
+	font-size: 1rem;
+`;
+
+const StyledButton = styled.button`
+	float: right;
+	width: 44px;
+	height: 44px;
+	background: transparent;
+	border: 0;
+	margin: -16px -16px 0 0;
+	padding: 0;
+	cursor: pointer;
+`;
+
+class BaseYoastModal extends React.Component {
+	/**
+	 * The constructor.
+	 *
+	 * @param {Object} props The props to use.
+	 */
+	constructor( props ) {
+		super( props );
+	}
+
+	/**
+	 * Returns the rendered html.
+	 *
+	 * @returns {ReactElement} The rendered html.
+	 */
+	render() {
+		return (
+			<Modal
+				isOpen={ this.props.isOpen }
+				onRequestClose={ this.props.onClose }
+				role="dialog"
+				contentLabel={ this.props.modalAriaLabel }
+				overlayClassName={ `${ this.props.className } yoast-modal__overlay` }
+				className={ `${ this.props.className } yoast-modal__content` }
+				appElement={ this.props.appElement }
+			>
+				{
+					this.props.heading &&
+						<StyledHeading className="yoast-modal__title">{ this.props.heading }</StyledHeading>
+				}
+				{
+					this.props.closeIconButton &&
+						<StyledButton
+							onClick={ this.props.onClose }
+							className={ `${ this.props.xCloseButtonClassName } yoast-modal__button-close-icon` }
+							aria-label={ this.props.closeIconButton }
```

Comment:
shouldn't this be `ariaLabel`?

Reply:
Isn't it the same thing? As far as I know, React accepts all the `aria-*` attributes as props.